README.md

```diff
@@ -29,4 +29,48 @@ To build and run this application from Docker:
 docker build -t radarbase/radar-rest-source-auth-backend:1.0.1 .
 
 docker run -p 8080:8080 radarbase/radar-rest-source-auth-backend:latest
-```
\ No newline at end of file
+```
+
+## Validation
```

Comment:
Could you please add documentation for setting the env variable "AUTHORIZATION_CODE"